conf/test.config

```diff
@@ -22,8 +22,8 @@ params {
     // Input data
     // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
     // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
+    input = 'https://github.com/nf-core/test-datasets/raw/eager/testdata/Mammoth/samplesheet.tsv'
```

Comment:
does it actually work without the `raw.`...?

Reply:
Yup! Apparently!